build/filter.go

```diff
@@ -909,3 +909,22 @@ func (in *InterfaceRemoveUuidFilter) Filter(profile *v1.HostProfile, host *v1.Ho
 	}
 	return nil
 }
+
+// InterfaceRemoveKernelFilter defines a profile and host filter that removes
+// kernel values from interfaces.
+type InterfaceRemoveKernelFilter struct {
```

Comment:
KernelFilter or HostKernelFilter, the "interface" term is a bit misleading

Reply:
done